app/lib/actions/katello/product/content_create.rb

```diff
@@ -43,6 +43,15 @@ def finalize
           repository = ::Katello::Repository.find(input[:repository_id])
           repository.content_id = input[:content_id]
           repository.save!
+
+          content = ::Katello::Content.create!(name: repository.name,
```

Comment:
@jlsherrill I opted to add the DB operations in the finalize method because I needed access to the Candlepin Content ID (repository.content_id). Trying to do this in the plan phase didn't seem possible. Accessing content_create.output[:response][:id] would not give the actual value in plan, but some other kind of Dynflow object...